src/style.css

```diff
@@ -1 +1,87 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+:root {
+  --main-color: #00acdc;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+.nav {
+  margin-left: auto;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  text-transform: uppercase;
+  height: 60px;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+.list-item {
+  justify-content: space-around;
+  font-family: inherit;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.list-item:not(:first-child) {
+  padding-left: 20px;
+}
+
+.link {
+  padding: 23px 0;
+  text-decoration: none;
+  color: black;
+  position: relative;
+}
+
+.link:hover {
+  text-decoration: none;
+  color: var(--main-color);
+}
+
+.is-active {
+  color: var(--main-color);
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  height: 4px;
+  width: 100%;
+  background: var(--main-color);
+  border-radius: 8px;
+  margin: 0 auto;
+  left: 0;
+  right: 0;
+  bottom: 0;
+}
+/* 
+a:active {
```

Comment:
do not leave comemnts

Reply:
deleted

Reply:
what would be a good way to reset paddings and margins?

Reply:
should it be done by linking extra reset.css file?

Reply:
@nadiia-kurovska It is better to move reset styles to separate `reset.css` file. The best way is to directly reset them for every needed element (p, h1, h2 etc.), avoiding `*` selector. There are also a lot of ready solutions (like [this](https://meyerweb.com/eric/tools/css/reset/)). There are also other ways of handling default CSS styles (e.g normalize).
You can read about them here: https://www.webfx.com/blog/web-design/css-reset-stylesheets/